scripts/tag-based-pre-processing/1-make_gene_matrix-alevin.R

```diff
@@ -11,14 +11,39 @@
 #        directory is specified, alevinQC is not run. (Optional)
 # "-l" - Optional label to add to output files. Generally necessary if processing
 #        multiple datasets in the same pipeline.
-#
+# "-f" - Optional but suggested filter application. If
```

Comment:
I would also have this be the last thing you talk about (i.e., move `-r` up) so it naturally leads into the `Filter options` section.

Reply:
It looks like I accidentally repeated this option twice. Going to combine instructions but underneath `Filter options`